### PR TITLE
updating setup.py

### PR DESCRIPTION
**Pull request by jarulraj**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,3 +1,140 @@
...
+    "sqlalchemy==1.3.20",
+    "sqlalchemy-utils==0.36.6",
+    "pyspark==3.0.2",
+    "petastorm==0.9.8",
+    "antlr4-python3-runtime==4.8",
+    "pyyaml==5.1",
+    "pymysql==0.10.1",
+]
+
+formatter_libs = ["black", "isort"]
```
Comment (gaurav274) [inline setup.py:63]: FLAKE8_VERSION_REQUIRED = "3.9.1"
BLACK_VERSION_REQUIRED = "22.6.0"
ISORT_VERSION_REQUIRED = "5.10.1"

File: setup.py [hunk 2]
```diff
@@ -1,3 +1,140 @@
...
+    packages=find_packages(exclude=[
+        "tests", 
+        "tests.*"
+    ]),
+    # https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html#the-console-scripts-entry-point
+    entry_points={"console_scripts": [
+        "eva_server=eva.eva_server:main",
+        "eva_client=eva.eva_cmd_client:main"
+    ]},
+    python_requires=">=3.8",
```
Comment (gaurav274) [inline setup.py:135]: We can do `>=3.7`

**Review by gaurav274 — APPROVED** [review]

File: setup.py (hunk 2 above)
Comment (jarulraj) [reply setup.py]: Resolved.

File: setup.py (hunk 1 above)
Comment (jarulraj) [reply setup.py]: Done

**Pull request merged**